graphql/resolver.go

```diff
@@ -35,6 +36,54 @@ func New(apiURL string) Config {
 			sc: dbConnector,
 		},
 	}
+	c.Directives.RequireHostAccess = func(ctx context.Context, obj interface{}, next graphql.Resolver, access HostAccessLevel) (interface{}, error) {
+		args, isStringMap := obj.(map[string]interface{})
+		if !isStringMap {
+			return nil, ResourceNotFound.Send(ctx, "host not specified")
+		}
+		hostId, hasHostId := args["hostId"].(string)
+		hostIdsInterface, hasHostIds := args["hostIds"].([]interface{})
+		if !hasHostId && !hasHostIds {
+			return nil, ResourceNotFound.Send(ctx, "host not specified")
+		}
+
+		hostIds := []string{}
+		if hasHostIds {
+			for _, v := range hostIdsInterface {
+				hostIds = append(hostIds, v.(string))
+			}
+		}
+		hostIdsToCheck := []string{hostId}
+		hostIdsToCheck = append(hostIdsToCheck, hostIds...)
```

Comment:
Could the same effect be achieved by:
```suggestion
		hostIdsToCheck := []string{hostId}
		if hasHostIds {
			for _, v := range hostIdsInterface {
				hostIdsToCheck = append(hostIdsToCheck, v.(string))
			}
		}
```